source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst

```diff
@@ -54,6 +54,7 @@ What's new
   - Power Management: :ref:`I/O Only Plus DDR <pm_io_only_plus_ddr>` mode
   - TI Apps Launcher: New Wi-Fi and Terminal Demo - :ref:`TI Apps Launcher <TI-Apps-Launcher-User-Guide-label>`
   - Snagfactory Support - :ref:`Snagfactory Tool <Flash-via-Fastboot>`
+  - Support for M2 CC33xx cards on Debian - :ref:`How to Enable M.2-CC33x1 in Linux <enable_m2cc3301>`
```

Comment:
There's an issue in using ref here.

Apparently you wanted to highlight Debian specific addition in Release notes, but as this Release note section is default built for linux (i.e Yocto), the changes w.r.t. `CONFIG_sdk in ('DebianSDK')` wont be picked - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/source/linux/How_to_Guides/Target/How_To_Enable_M2CC3301_in_linux.rst?plain=1#L43

One option I see is to point to Debian SDK docs as a external hyperlink (similar to MCU+SDK links)

Reply:
@StaticRocket The reason we have a different "Release Notes" section for Android is because its OPN is different. But Debian and Buildroot are clubbed with the Yocto distribution, which is why we are maintaining the same notes.

Reply:
@StaticRocket Yeah, we had a internal discussion with @khasim-syed on it & the suggestion was to use the existing Release Notes to highlight Debian, Buildroot as well since it's a common OPN (https://www.ti.com/tool/download/PROCESSOR-SDK-LINUX-AM62X/10.00.07.04) for delivering artifacts of all three distros

Reply:
@jsuhaas22 IMO, you can go ahead with external hyperlinks for this as well similar to https://github.com/TexasInstruments/processor-sdk-doc/pull/144